client/src/pages/Dashboard/components/SurveyCard.module.css

```diff
@@ -21,4 +33,4 @@
 
 .ml {
   margin-left: 1rem;
-}
\ No newline at end of file
+}
```

Comment:
I thought we were making `makeStyles` in a `styles.js` file as standard styling.

Reply:
actually we've talked about using both styles.js, module.css and in-line styling, due to the complexity of using a UI library!